### PR TITLE
Feature/custom tx timestamp

### PR DESCRIPTION
**Pull request by pengin7384**
Done
- Provide transaction creation with user defined timestamp

### CODE REVIEW

File: src/ain.ts [hunk 1]
```diff
@@ -222,7 +222,8 @@ export default class Ain {
     if (nonce === undefined) {
       nonce = await this.getNonce({address, from: "pending"}) + 1;
     }
-    return Object.assign(tx, { nonce, timestamp: Date.now() });
+    const timestamp = !transactionInput.timestamp ? Date.now() : transactionInput.timestamp;
```
Comment (platfowner) [inline src/ain.ts:225]: minor: what about true case first? = transactionInput.timestamp ? transactionInput.timestamp : Date.now()?

Comment (pengin7384) [reply src/ain.ts]: Looks better. Done!

**Comment by liayoo** [comment]
If you're planning to release a new version after merging this pr, could you increase the package version in package.json to something like 1.0.3?

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by pengin7384** [comment]
@liayoo Thanks. Done!

**Review by liayoo — APPROVED** [review]
LGTM. Thanks for the new feature!

**Comment by pengin7384** [comment]
Thanks for reviews

**Pull request merged**